lib/storage/providers/SQLiteStorage.js

```diff
@@ -66,7 +66,7 @@ const provider = {
     multiSet(pairs) {
         const stringifiedPairs = _.map(pairs, pair => [
             pair[0],
-            JSON.stringify(pair[1]),
+            JSON.stringify(pair[1] || null),
```

Comment:
Let's only short circuit to `null` in the case of an explicitly `undefined` value? Anything falsy will default to `null` with what you have here. It might be fun for a while, but will probably lose it's charm quickly 😄

Reply:
Oh, good call.